scripts/lint.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+set -e
+
+poetry run pylint $(git ls-files '*.py')
```

Comment:
smart solution

Reply:
yeah, john is really smart, had to scroll all the way to the third answer: https://stackoverflow.com/a/63044665